<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatrixImageView</class>
 <widget class="QWidget" name="MatrixImageView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MatrixImageView</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="1">
    <widget class="QGroupBox" name="contrastGroupBox">
     <property name="title">
      <string>Kontrast</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QSpinBox" name="contrastSpinBoxIntensity"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QGroupBox" name="structureGroupBox">
     <property name="title">
      <string>Struktur</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Distanz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="structureSpinBoxDistance"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Sigma</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="structureSpinBoxSigma"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="checkHistoEqualize">
     <property name="text">
      <string>Histo Equalize</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Schärfen</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Sigma</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Stärke</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_2"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Schwellenwert</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_3"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="claheGroupBox">
     <property name="title">
      <string>CLAHE</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="claheLabelClipBox">
        <property name="text">
         <string>Größe (Size)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="claheSpinBoxSize"/>
      </item>
      <item>
       <widget class="QLabel" name="claheLabelSizeBox">
        <property name="text">
         <string>Stärke (Clip)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="claheSpinBoxClip"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" rowspan="13">
    <widget class="MatrixFrameView" name="frameView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBox"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MatrixFrameView</class>
   <extends>QFrame</extends>
   <header>MatrixFrameView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
